package.json

```diff
@@ -35,43 +37,45 @@
     "validation"
   ],
   "peerDependencies": {
-    "@remix-run/react": "^1.0.0",
-    "@remix-run/server-runtime": "^1.0.0",
+    "@remix-run/react": "^1.0.6",
+    "@remix-run/server-runtime": "^1.0.6",
     "react": "^17.0.2"
   },
   "devDependencies": {
-    "@remix-run/react": "^1.0.0",
-    "@remix-run/server-runtime": "^1.0.0",
+    "@remix-run/react": "^1.0.6",
+    "@remix-run/server-runtime": "^1.0.6",
     "@types/jest": "^27.0.3",
-    "@types/react": "^17.0.36",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
+    "@types/lodash": "^4.14.178",
+    "@types/react": "^17.0.37",
+    "@typescript-eslint/eslint-plugin": "^5.6.0",
+    "@typescript-eslint/parser": "^5.6.0",
     "babel-eslint": "^10.1.0",
-    "eslint": "^7.32.0",
+    "eslint": "^8.4.1",
     "eslint-config-react-app": "^6.0.0",
     "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-flowtype": "^5.10.0",
+    "eslint-plugin-flowtype": "^8.0.3",
     "eslint-plugin-import": "^2.25.3",
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
     "fetch-blob": "^3.1.3",
     "husky": "^7.0.4",
-    "jest": "^27.3.1",
+    "jest": "^27.4.4",
     "lint-staged": "^12.1.2",
-    "prettier": "^2.5.0",
+    "prettier": "^2.5.1",
     "react": "^17.0.2",
-    "ts-jest": "^27.0.7",
-    "typescript": "^4.5.2",
+    "ts-jest": "^27.1.1",
+    "typescript": "^4.5.3",
     "yup": "^0.32.11",
     "zod": "^3.11.6"
   },
   "dependencies": {
+    "lodash": "^4.17.21",
```

Comment:
The thing is that the `flatten` and `unflatten` functions depends on several functions from `lodash`. I will change to single imports for each function to avoid bundling the entire `lodash` library. This will have the exact same impact than copying the `lodash` implementations to the code.